app/views/devise/sessions/new.html.haml

```diff
@@ -0,0 +1,8 @@
+.login
+  - unless flash[:alert].blank?
+    .bs-callout.bs-callout-warning
+      %h4= flash[:alert]
```

Comment:
@vlasiak, I suggest to introduce a view object, here is [a sample](https://gist.github.com/sergeyradkevich/92160f78b9a61016a9ad) and use its interface instead of a hash.  
Just a heads up: [view objects](https://github.com/vlasiak/rails_tada/pull/6#issuecomment-119472980).